rust/src/disassembly.rs

```diff
@@ -307,10 +307,9 @@ impl std::fmt::Display for DisassemblyTextLine {
 }
 
 impl From<Vec<InstructionTextToken>> for DisassemblyTextLine {
-    fn from(mut tokens: Vec<InstructionTextToken>) -> Self {
-        tokens.shrink_to_fit();
+    fn from(tokens: Vec<InstructionTextToken>) -> Self {
+        let mut tokens: Box<[_]> = tokens.into();
```

Comment:
```suggestion
        let mut tokens = tokens.into_boxed_slice();
```
You can use `Vec::into_boxed_slice` to avoid having to annotate the type here.